### ISSUE TITLE
Failed to get storage info: x509 certificate signed by unknown authority

### ISSUE THREAD

**Issue by trexx** [minio/operator#1186]
Every 3 minuities the operator attempts to establish a connection to `/minio/admin/v3/storageinfo` and fails due to `x509: certificate signed by unknown authority`. 

We're using a custom certificate for Minio generated by Cert-manager. The secret is declared in the Tenant resource, via `externalCertSecret` which contains the full chain (ca.crt, tls.crt, tls.key). This seems to be working fine for the most part, however the operator is not able to populate usage metrics in the tenant resource, resulting in no usage data in the console. 
There is clearly some communication established between the operator and the Minio tenant as the status fields are populated with cluster health but this is probably likely due to the operator [disabling certificate verification](https://github.com/minio/operator/blob/v4.4.25/pkg/controller/cluster/monitoring.go#L303) for those checks.

Following some [documentation](https://github.com/minio/operator/blob/master/docs/tls.md#using-your-own-ca-certificate-for-minio-operator), I've created a Opaque secret called `operator-ca-tls` with a `public.crt` key containing our root ca certificate but this makes no difference (The documentation suggests the key should be `ca.crt` however this causes the operator to complain).
Alternatively the helm chart offers no way of mounting extra volumes / volume mounts to overwrite the containers trust bundle.

## Expected Behavior
The operator expands its container TLS trust bundle with the contents of externalCertSecret which has the whole certificate chain (ca.crt, tls.crt, tls.key). ca.crt contains a single PEM certificate, tls.crt contains an intermediary ca cert and the server certificate.

## Current Behavior
`1 monitoring.go:178] 'minio-infra/infra' Failed to get storage info: Get "https://minio.minio-infra.svc.cluster.local/minio/admin/v3/storageinfo": x509: certificate signed by unknown authority`
No usage metrics in Console

## Possible Solution
N/A

## Context
The Console is not displaying usage metrics as the operator is unable to fetch them from the tenant for storage in the Tenant resource.

## Regression
Not that I know of

## Your Environment
* Version used (`minio-operator`): 4.4.25
* Environment name and version (e.g. kubernetes v1.17.2): Kubernetes 1.23.4
* Server type and version: Official Minio Docker image RELEASE.2022-06-20T23-13-45Z

**Comment by sheromon** [comment]
I have the same or a similar problem. I've defined both `externalCertSecret` and `externalCaCertSecret` in my tenant spec, and I get error messages from `monitoring.go` that say `Failed to get storage info: Get "https://minio.svc.cluster.local/minio/admin/v3/storageinfo": x509: certificate signed by unknown authority`. Any requests sent to the minio api get a 503 "Service Temporarily Unavailable" in response as well.

**Comment by trexx** [comment]
This is what we did to get past that error.
Create a secret in the Minio Operator namespace named "operator-ca-tls" with a key "ca.crt" containing your ca certificate.
Documentation for this is https://github.com/minio/operator/blob/master/docs/tls.md#using-your-own-ca-certificate-for-minio-operator

Although I did that before creating this issue, the second time around I might have killed / restarted the operator for it to pick it up.
